### ISSUE TITLE
Please target Android 13 or higher

### ISSUE THREAD

**Issue by jcperil** [forkgram/TelegramAndroid#242]
At this point, Android 13 and 14 has been here for a while and Android 15 is already by the corner. Can you please target Android 13 or higher?

**Comment by shamefulCake1** [comment]
No, please, no. I still have Android 9 on several phones, and my main device is Android 6.

**Comment by 23rd** [comment]
> my main device is Android 6.

Oh, sounds interesting, how it is going on?

**Comment by 23rd** [comment]
> At this point, Android 13 and 14 has been here for a while and Android 15 is already by the corner. Can you please target Android 13 or higher?

I don’t really understand what do you expect from me. A few weeks ago I have got device with 14 android on board (not as my primary one) and my fork works as normal on this device. So, what’s the problem?

**Comment by shamefulCake1** [comment]
> > my main device is Android 6.
> 
> Oh, sounds interesting, how it is going on?

Works well, quite fast. No problem at all.

**Comment by flexxxxer** [comment]
TLDR: already done at [September 10 2024](https://github.com/forkgram/TelegramAndroid/blob/3d718c8d36b0d0eb521c22c09bbc23a4aa3d98d1/TMessagesProj/build.gradle) - upgraded to 34 (A14) and [September 1 2025](https://github.com/forkgram/TelegramAndroid/blob/aa5fe2ee8c9711e0a615cc955fba0ca040896144/TMessagesProj/build.gradle) - upgraded to 35 (A15). All of this due to upstream changes.

Table of Android versions and their API levels: https://apilevels.com
This can be closed @23rd
